imports/plugins/included/product-variant/containers/variantFormContainer.js

```diff
@@ -49,52 +49,8 @@ const wrapComponent = (Comp) => (
 
         return validationStatus;
       }
-    }
-
-    isProviderEnabled = () => {
-      const shopId = Reaction.getShopId();
 
-      const provider = Packages.findOne({
```

Comment:
So this provided functionality to create a dropdown with available taxcodes that are valid for the tax provider. Are we not allowing that now?

Reply:
It still works, just in a different way. There's a new GraphQL query that is called to get the list of available tax codes, which then triggers showing the Select if a list is returned.